Week-1/Readme.md

```diff
@@ -0,0 +1,58 @@
+## What is the main funtionality of the Browser?
+
+A web browser, is a software application that you use to browse the World Wide Web. It works by sending a request to a web server and then receives a response 
+containing the content that is to be displayed on the screen of your device. Once the browser is open on your device, there is the address bar where you input the 
+address of the website that you want to visit. The address is commonly known as the Uniform Resource Locator or URL for short. The URL contains the protocol or the 
+HTTP, the domain name, usually the name of the website, and the file path, or the path to the page that is displayed. When you make a request using this URL, 
+the browser and server communicate using a protocol known as the Hypertext Transfer Protocol or HTTP. Once the web browser receives the content, it displays it on the 
```

Comment:
@porushmarwaha you could have explained more on diff parts of URI and a lil about DNS